### DISCUSSION TITLE
How to add Akash Network to Keplr mobile Wallet

### DISCUSSION THREAD

**Discussion by suect1** [akash-network/community#238] [Q&A]
Hello,

How can I add Akash Network to Keplr mobile wallet?

Thank you

**Comment by brewsterdrinkwater** [comment]
This would be a great question to ask in the [Akash Network Discord](https://discord.com/invite/akash). If you ask in "general-chat" someone from the Akash community can answer this question in real time, and walk you through it, if needed. 

Here is some documentation I found: 

- https://help.keplr.app/articles/create-a-new-wallet-with-keplr-mobile
- https://docs.akash.network/tokens-and-wallets/keplr

**Reply by suect1** [reply]
Hello,

Thank you for your reply.
According to Keplr is up to Akash to request mobile integration.

**Comment by bmenzalji** [comment]
We pinged Keplr's team, it doesnt look like AKT is supported on mobile at this time.  I've reached out to understand the requirements there and will follow up once there's a plan.

In the meantime, please use the web app for Keplr.  I think Leap Mobile though has AKT support on their wallet

https://www.leapwallet.io/

**Comment by suect1** [comment]
Hello,

Thank you for your email and clarification.



<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by 88plug** [comment]
Cosmostation mobile wallet supports Akash (AKT).

[Cosmostation Google Play Store](https://play.google.com/store/apps/details?id=wannabit.io.cosmostaion&hl=en_US&gl=US&pli=1)
and
[Cosmostation Apple App Store](https://apps.apple.com/us/app/cosmostation/id1459830339)

**Comment by Fardonal** [comment]
Hello! Can you please tell me if leapwallet is as reliable and safe as keplr? Thank you!

**Reply by bmenzalji** [reply]
Leap wallet is as reliable as Cosmostation/Keplr, so yes.  In terms of safety, that's on you to safe guard your passphrase if youre using a hot wallet, or using a ledger and take similar precautions @Fardonal 

For all other safety questions, reach out to Leap Wallet in their TG group to ask them about their security and you can assess whether or not it's as safe as Keplr :) 

Leap Wallet does have mobile wallet support for AKT, whereas Keplr does not at this time.

**Comment by bmenzalji** [comment]
Keplr now supports Akash in their mobile wallet.  Please check their updates for more information.

**Comment by suect1** [comment]
Hello,

Thank you for your email.
Very appreciated
